sos/plugins/python.py

```diff
@@ -33,9 +35,11 @@ class RedHatPython(Python, RedHatPlugin):
 
     packages = ('python', 'python36', 'python2', 'platform-python')
     option_list = [
-        ('gather_py_hashes', "gather hashes for all python files", 'slow',
+        ('hashes', "gather hashes for all python files", 'slow',
          False)]
 
+    python_version = "platform-python"
```

Comment:
@bmr-cymru you told to modify it to `platform-python`, do I correctly understand?

Reply:
This needs to have the same logic as the current plugin (so it needs to be in the `setup()` method):

```python
        if isinstance(self.policy, sos.policies.redhat.RHELPolicy) and \
                self.policy.dist_version() > 7:
            self.python_version = "/usr/libexec/platform-python -V"
        super(RedHatPython, self).setup()
```

Then the main `setup()` method in the `Python` class needs to `self.add_cmd_output(self.python_version, suggest_filename='python-version')`.

That way, the `python-version` file will always use the correct command for the distro it is running on, and the `hashes` option will be available to all distributions.